gitpatches/Meter.c.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/Meter.c b/Meter.c
+index 7f634f19..fc3025b1 100644
+--- a/Meter.c
++++ b/Meter.c
+@@ -315,6 +315,7 @@ static void GraphMeterMode_draw(Meter* this, int x, int y, int w) {
+    x += captionLen;
+    w -= captionLen;
+ 
++#ifndef __MVS__
+    if (!timercmp(&pl->realtime, &(data->time), <)) {
```

Comment:
Wonder if we should add an assert here in case we encounter it in runtime and we get a hard to debug issue.